adam-core/src/main/scala/org/bdgenomics/adam/rdd/variant/VCFInFormatter.scala

```diff
@@ -56,7 +59,8 @@ private[variant] case class VCFInFormatter private (
   protected val companion = VCFInFormatter
 
   // make a converter
-  val converter = new VariantContextConverter(Some(sequences))
+  val converter = new VariantContextConverter(headerLines,
+    ValidationStringency.LENIENT)
```

Comment:
Ideally, in the long run we'll come up with a good way to pass validation stringency to the pipe code, probably through a Hadoop config value. In the absence of a way to override the default of strict, which will fail on any validation error (which are myriad), I prefer lenient, which will drop any erroneous variants/genotypes, but which will log the record that was dropped and why it was dropped.